package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git@github.com:microsoft/react-native.git"
+    "url": "git@github.com:microsoft/react-native-macos.git"
```

Comment:
Also missed updating this.   This should fix up the return repo url on https://www.npmjs.com/package/react-native-macos